src/core/startCoreFactory.js

```diff
@@ -35,6 +35,7 @@ function startCoreFactory(
       'dashd',
       '-conf=/dash/.dashcore/dash.conf',
       `-port=${config.get('core.p2p.port')}`,
```

Comment:
we have these options in dashd config I suppose

Reply:
Yes, these options have been removed here.